From 610f57794f059b6f8591ac24fbb6fadeebf341cb Mon Sep 17 00:00:00 2001
From: Jonathan M Hsieh <jon@cloudera.com>
Date: Fri, 5 Feb 2016 07:44:04 -0800
Subject: [PATCH 195/217] CLOUDERA-BUILD change pom to use hbase
 1.2.0_cdh5.7.0-SNAPSHOT, spark
 1.6.0_cdh5.7.0-SNAPSHOT

Change-Id: Ia7cac35bc1dd0273d49ff55b2f8c164eb50ca5aa
Reason: New Feature
Author: Jonathan Hsieh
Ref: CDH-35578

Change-Id: I6c2d479a2b7032e40e9048adde400ef1dc975671
---
 hbase-spark/pom.xml |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/hbase-spark/pom.xml b/hbase-spark/pom.xml
index e48f9e8..75d5cb4 100644
--- a/hbase-spark/pom.xml
+++ b/hbase-spark/pom.xml
@@ -30,14 +30,14 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>2.0.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.7.0-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-spark</artifactId>
     <name>Apache HBase - Spark</name>
 
     <properties>
-        <spark.version>1.3.0</spark.version>
+        <spark.version>${cdh.spark.version}</spark.version>
         <scala.version>2.10.4</scala.version>
         <scala.binary.version>2.10</scala.binary.version>
         <top.dir>${project.basedir}/..</top.dir>
@@ -569,4 +569,4 @@
         </plugins>
     </build>
 
-</project>
\ No newline at end of file
+</project>
-- 
1.7.9.5

